core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendConf.scala

```diff
@@ -233,4 +234,7 @@ object ExternalBackendConf {
 
   /** Disable version check of external H2O backend */
   val PROP_EXTERNAL_DISABLE_VERSION_CHECK = ("spark.ext.h2o.external.disable.version.check", false)
+
+  /** Name of file where flatfile is stored in case of automatic mode of external backend */
+  val PROP_EXTERNAL_FLATFILE_NAME = ("spark.ext.h2o.external.flatfile.name", None)
```

Comment:
Just a question: Doesn't the preporty describe a relative path of a flat file and not just name?

Reply:
Nope, just a name

Reply:
Ok, thanks!